src/window.ts

```diff
@@ -90,8 +90,9 @@ export type SMonsterEventLive = {
   /** Is the monster currently available? */
   live: true;
 
-  x: number;
-  y: number;
+  /** NOTE: Some event monsters don't have x and y (e.g.: Slenderman) */
+  x?: number;
+  y?: number;
```

Comment:
Maybe we should have the type depends on the monster. Since we'll know what monsters have what infos in their S event, we could be 100% precise and omit`x/y/end` when relevant.